pack/aoa_encounter.json

```diff
@@ -2641,6 +2641,7 @@
         "quantity": 1,
         "set_code": "overseer",
         "set_position": 3,
+        "text": "Victory 5.\nCannot take damage while another minion is at the mission.\n<b>Mission Response</b>: After you discard cards, attach each card with a resource ([wild]) discarded to Abyss facedown.<i>(They cannot be used for the mission attempt.)</i>",
```

Comment:
A couple of minor things here:

1. I think this is missing the `wild` word in
```
...each card with a wild resource...
```

2. I think there should be a space before the opening `<i>` tag
```
...Abyss facedown. <i>(They...
```

Reply:
> A couple of minor things here:

nice catch, thanks